packages/functional-tests/tests/oauth/forceAuth.spec.ts

```diff
@@ -21,59 +29,82 @@ test.describe('severity-1 #smoke', () => {
       await relier.clickForceAuth();
 
       // Email is prefilled
-      await expect(await login.page.innerText('#prefillEmail')).toContain(
+      expect(await login.page.innerText('#prefillEmail')).toContain(
```

Comment:
We can use it and I will replace. My fingers are itching to replace it properly with a built-in locator and auto-awaiting assert, but I will refrain since plans for the login POM are undefined atm.